recon/src/protocol.rs

```diff
@@ -840,19 +835,18 @@ where
         for key in keys {
             if let Some(value) = self.recon.value_for_key(key.clone()).await? {
                 self.stream
-                    .feed((self.value_resp_fn)(
+                    .send((self.value_resp_fn)(
```

Comment:
use `send_all` here instead?

Reply:
sticking with `send` until (if) we change the retrieval logic to work on many keys.. since we're yielding to find another key, we should send what we have